src/services/api/handlers/deletePost.ts

```diff
@@ -1,25 +1,29 @@
+import { z, ZodError } from "zod";
 import { response } from "../libs/handler";
+import { post } from "../models/Post";
+import { PostService } from "../services/postService";
 
 export const deletePost = async ({ pathParameters }) => {
   try {
-    if (!pathParameters || !pathParameters.id) {
-      return response({
-        statusCode: 400,
-        body: { message: "Invalid request" },
-      });
-    }
+    const validParams = z.object({
+      id: z.string().uuid(),
+    });
+
+    const params = validParams.parse(pathParameters);
 
-    const { id } = pathParameters;
+    const postToDelete = await new PostService(post).deletePost(params.id);
 
     return response({
       statusCode: 200,
-      body: { post: `Post with ${id} was deleted` },
+      body: postToDelete,
```

Comment:
should we just return a 204 and no body?

Reply:
@benjaminpaige 

No we should just leave it, don't try to be clever